documentation/reference/!plugin-template.md

```diff
@@ -0,0 +1,82 @@
+---
+title: {Plugin Name}
+---
+<!-- template preamble -->
+
+## Template preamble (remove!)
+
+Reference articles concisely present information as a structured set of entries,
+with little to no procedural or instructional content. They are like the
+nutritional information on a food package.
+
+Alongside other one-off reference page types, we will have a collection of Kong
```

Comment:
how do we decide when to write a reference page vs just refer to existing Kong content?

Reply:
When either a) it is a custom plugin, not from Kong, or b) it is a commonly used plugin (from Kong).
In the case of b), as the template shows, the reference page should be quite brief, draw from Kong docs, and omit some sections and instead reference Kong docs